### DISCUSSION TITLE
[New Feature] Update the Caret Position of the Active Note when Selecting a Note in GA

### DISCUSSION THREAD

**Discussion by FelipeRearden** [SkepticMystic/graph-analysis#37] [Ideas]
Hello @SkepticMystic !!!!!

I don’t know the words in English to explain but I’m gonna try to explain what I’m 
suggesting.

When opening a note from the GA side panel would be amazing if the caret position of the “new” active note could jump to the `[[note]]` clicked in the GA;
- Place the caret position in the line of the yellow (highlight) position;

By default the caret position is placed in `Line 1` instead. 

Let’s using the Joe Biden example:

For the Joe Biden note I have [[Michele Obama]] two times in the [[Federal Reserve]] note 

![3](https://user-images.githubusercontent.com/85180696/141651721-edca540b-ed4a-4023-b6e5-fb4f4ce57216.png)

Step 1: Open Joe Biden note
Step 2: Open GA 
Step 3: Frozen
Step 4: Open Michelle Obama arrow
Step 5: cmd/ctrl click in the first [[Michelle Obama]] note in the [[Federal Reserve]] note 

![4](https://user-images.githubusercontent.com/85180696/141651727-6bfccc8d-14ab-4fe6-8f0a-44ee477b62a1.png)

As you can see: 
- The note will open;
- Yellow highlight [[Michelle Obama]]  I didn’t get the yellow when doing the print screen :(
- The caret position is in Line 1 instead of Line 17 :( 

Would be great if the caret position go to the line related to the selection in GA!

This could be a great feature for long notes (300 to 600 lines) to fast find the exact point of interest in the note:
- It’s a behavior pretty similar to what we can achieve when using our beloved plug-in Advanced Cursors :)

---

I don’t know if is a feasible one :) 

Thanks for reading this :)

Have a great day!

**Comment by SkepticMystic** [comment]
Ahh great idea! That's definitely doable :)

**Reply by FelipeRearden** [reply]
> Ahh great idea! That's definitely doable :)

AMAZING !!!!! I knew that you would like this one @SkepticMystic !!!!

😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 😄 

Thanks !!!!!!!!!
